src/test/java/io/redis/examples/CmdsGenericExample.java

```diff
@@ -0,0 +1,434 @@
+
+
+        // STEP_START object_encoding
+
+        // STEP_END
+
+        // Tests for 'object_encoding' step.
+        // REMOVE_START
+
+        // REMOVE_END
+
+
+        // STEP_START object_freq
+
+        // STEP_END
+
+        // Tests for 'object_freq' step.
+        // REMOVE_START
+
+        // REMOVE_END
+
+
+        // STEP_START object_idletime
+
+        // STEP_END
+
+        // Tests for 'object_idletime' step.
+        // REMOVE_START
+
+        // REMOVE_END
+
+
+        // STEP_START object_refcount
+
+        // STEP_END
+
+        // Tests for 'object_refcount' step.
+        // REMOVE_START
+
+        // REMOVE_END
+
+
+        // STEP_START persist
+
+        // STEP_END
+
+        // Tests for 'persist' step.
+        // REMOVE_START
+
+        // REMOVE_END
+
+
+        // STEP_START pexpire
+
+        // STEP_END
+
+        // Tests for 'pexpire' step.
+        // REMOVE_START
+
+        // REMOVE_END
+
+
+        // STEP_START pexpireat
+
+        // STEP_END
+
+        // Tests for 'pexpireat' step.
+        // REMOVE_START
+
+        // REMOVE_END
+
+
+        // STEP_START pexpiretime
+
+        // STEP_END
+
+        // Tests for 'pexpiretime' step.
+        // REMOVE_START
+
+        // REMOVE_END
+
+
+        // STEP_START pttl
+
+        // STEP_END
+
+        // Tests for 'pttl' step.
+        // REMOVE_START
+
+        // REMOVE_END
+
+
+        // STEP_START randomkey
+
+        // STEP_END
+
+        // Tests for 'randomkey' step.
+        // REMOVE_START
+
+        // REMOVE_END
+
+
+        // STEP_START rename
+
+        // STEP_END
+
+        // Tests for 'rename' step.
+        // REMOVE_START
+
+        // REMOVE_END
+
+
+        // STEP_START renamenx
+
+        // STEP_END
+
+        // Tests for 'renamenx' step.
+        // REMOVE_START
+
+        // REMOVE_END
+
+
+        // STEP_START restore
+
+        // STEP_END
+
+        // Tests for 'restore' step.
+        // REMOVE_START
+
+        // REMOVE_END
+
+
+        // STEP_START scan1
+
+        // STEP_END
+
+        // Tests for 'scan1' step.
+        // REMOVE_START
+
+        // REMOVE_END
+
+
+        // STEP_START scan2
+
+        // STEP_END
+
+        // Tests for 'scan2' step.
+        // REMOVE_START
+
+        // REMOVE_END
+
+
+        // STEP_START scan3
+
+        // STEP_END
+
+        // Tests for 'scan3' step.
+        // REMOVE_START
+
+        // REMOVE_END
+
+
+        // STEP_START scan4
+
+        // STEP_END
+
+        // Tests for 'scan4' step.
+        // REMOVE_START
+
+        // REMOVE_END
+
+
+        // STEP_START sort
+
+        // STEP_END
+
+        // Tests for 'sort' step.
+        // REMOVE_START
+
+        // REMOVE_END
+
+
+        // STEP_START sort_ro
+
+        // STEP_END
+
+        // Tests for 'sort_ro' step.
+        // REMOVE_START
+
+        // REMOVE_END
+
+
+        // STEP_START touch
+
+        // STEP_END
+
+        // Tests for 'touch' step.
+        // REMOVE_START
+
+        // REMOVE_END
```

Comment:
@andy-stark-redis Could you explain why you put those placeholders in each PR? 

If you need a way to check which steps are missing for any client, there is a better approach https://redis-stack.github.io/redis-stack-website/examples/

I built a script for the old Redis-stack website a while ago that parses and auto-generates this page as markdown and can help in setting it up for the new website.

Reply:
@uglide The only reason why the placeholders are there is because I'm using a tool that generates the boilerplate comments (`STEP_START xxx`, etc) for the whole set of examples all at once. The tool was originally designed for the tutorial pages (eg, [strings](https://redis.io/docs/latest/develop/data-types/strings/)) where it makes sense to add all the examples for a doc page together.

The approach we're using for the command pages is to batch related command examples together into one source file (eg, all the [generic group](https://redis.io/docs/latest/commands/?group=generic) commands). However, there are lots of commands, so we're adding them in small batches using a priority order based mainly on page traffic. So, the boilerplate tool adds placeholders for all the examples that will eventually live in the source file, but we only add the actual example code here and there in priority order.

If the placeholders are a problem then I'm happy to remove them and just add the boilerplate manually or modify the tool to do individual examples.

Reply:
Removed placeholders as requested.